packages/xo-server/src/xo-mixins/proxies.mjs

```diff
@@ -180,6 +180,7 @@ export default class Proxy {
     const { vmUuid } = await this._getProxy(id)
     const xapi = this._app.getXapi(vmUuid)
     await xapi.getObject(vmUuid).update_xenstore_data(xenstoreData)
+    this.getProxyApplianceUpdaterState(REMOVE_CACHE_ENTRY, id)
```

Comment:
Should we do this here or after the appliance has finished rebooted?

Reply:
IMHO, after performing the reboot to ensure that everything is in order.